File: setup_db.rb
#!/usr/bin/env ruby
require 'mysql2'

client = Mysql2::Client.new(:host => "localhost", :username => "root", :database => "test")

results = client.query("DROP TABLE IF EXISTS `user`")
results = client.query(<<EOS);
CREATE TABLE `user` (
  `id` varchar(32) NOT NULL,
  `name` varchar(256) NOT NULL,
  `mail` varchar(1024) NOT NULL,
  `c3` int(11) DEFAULT NULL,
  `c4` bigint(20) unsigned DEFAULT NULL,
  `c5` int(11) DEFAULT NULL,
   PRIMARY KEY (`id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8
EOS

(0..10000).each do |i|
  name = ((0..9).to_a + ("a".."z").to_a).sample(10).join
  mail = ((0..9).to_a + ("a".."z").to_a).sample(10).join + "@example.com"
  results = client.query("insert into `user` values (#{i},\"#{name}\",\"#{mail}\",0,0,0)");
end

File: python/setup_db.py
#!/usr/bin/env python
import string
import random
import MySQLdb

con = MySQLdb.connect('localhost', 'root', db='test')
cur = con.cursor()
cur.execute("DROP TABLE IF EXISTS `user`")
cur.execute("""\
CREATE TABLE `user` (
  `id` varchar(32) NOT NULL,
  `name` varchar(256) NOT NULL,
  `mail` varchar(1024) NOT NULL,
  `c3` int(11) DEFAULT NULL,
  `c4` bigint(20) unsigned DEFAULT NULL,
  `c5` int(11) DEFAULT NULL,
   PRIMARY KEY (`id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8""")

chars = string.ascii_lowercase + string.digits

for i in range(10001):
    name = ''.join(random.sample(chars, 10))
    mail = ''.join(random.sample(chars, 10)) + "@example.com"
    cur.execute("INSERT INTO `user` VALUES (%s, %s, %s, 0, 0, 0)", (i, name, mail))
con.commit()
